es/index.md

```diff
@@ -1,110 +1,111 @@
 ## Presentaciones listas para usar
 
-Esta es una colección de presentaciones que puedes usar, adaptar, y modificar para *tu* propia presentacion! Esta construida por Cloud Advocates de Microsoft en colaboración con Microsoft Student Ambassadors.
+¡Esta es una colección de presentaciones que puedes usar, adaptar, y modificar para *tu* propia presentación! Esta construida por Cloud Advocates de Microsoft en colaboración con los Microsoft Learn Student Ambassadors.
 
-🚀 Present in 3 easy steps:
+🚀 Presenta en 3 pasos sencillos:
+1. Encuentra un tema que está listo para usar de [Inteligencia Artificial/Aprendizaje Automático](#-ai-and-ml), [Datos](#-data-and-data-science), o [Desarrollo Web](#-web-development). Por ejemplo: [Introducción a Aplicaciones Web Estáticas](./short/intro-static-web-apps/README.md)
```

Comment:
Dejemos "Machine Learning" sin traducir. Creo que es mas comun usar ML / Machine Learning que Aprendizaje Automatico.

Y los links tienen que coincidir con los titulos. Como se cambio "AI and ML" entonces el link no puede ser `#-ai-and-ml` 

El ultimo link a static web apps no funciona porque es una ruta que no existe. Como este README esta en `/es/` ahora el link ya no puede ser `./short` si no `../short/`

Reply:
Okidoki, vere eso :)